src/preset.ts

```diff
@@ -467,24 +467,39 @@ export function setPresetsPlusIncluded(folder: string, presets: PresetsFile | un
     presetsPlusIncluded.set(folder, presets);
 }
 
-export function setUserPresetsPlusIncluded(folder: string, presets: PresetsFile | undefined) {
+export function setUserPresetsHelper(presets: PresetsFile | undefined) {
```

Comment:
Why did you make this a helper?

Reply:
oh nvm i guess you wanted a version without the `userPresetsPlusIncluded.set(folder, presets)` line